src/ControlSystem/Controller.cpp

```diff
@@ -46,7 +46,8 @@ DataVector Controller<DerivOrder>::operator()(
   }
 
   // compute denominator associated with correcting for time offset
-  DataVector denom{q_and_derivs[0].size(), 1.0};
+  DataVector& denom{tau};
```

Comment:
you want to move this

Reply:
Thanks guys. Rebased and squashed -- but, please let me know if there is anything else you'd like me to do